Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM node:16-slim
+COPY dist dist
+ENTRYPOINT [ "node", "/dist/index.js" ]
```

Comment:
We could even add the building stuff here and publish the actual docker image somewhere public :wink: But, that'd be for a next iteration.